app/src/main/res/values/strings.xml

```diff
@@ -242,41 +242,41 @@
     <string name="login_enter_encryption_details">Secret Encryption Password</string>
     <string name="login_encryption_account_label">Account:</string>
     <string name="login_service_details_description">This is your login password, *not* your encryption password!</string>
-    <string name="login_forgot_password">Forgot password?</string>
+    <string name="login_forgot_password">Forgot your password?</string>
 
-    <string name="login_wrong_username_or_password">Couldn\'t authenticate (probably wrong username or password).\nHave you registered?</string>
+    <string name="login_wrong_username_or_password">Could not log in (probably wrong username or password).\nHave you registered?</string>
     <string name="login_view_logs">View logs</string>
 
     <string name="setting_up_encryption">Setting up encryption</string>
     <string name="setting_up_encryption_content">Please wait, setting up encryption…</string>
 
-    <string name="account_creation_failed">Account creation failed</string>
+    <string name="account_creation_failed">Could not create account</string>
     <string name="wrong_encryption_password">Wrong encryption password</string>
-    <string name="wrong_encryption_password_content">Got an integrity error while accessing your account, which most likely means you put in the wrong encryption password.\nPlease note that the username is case sensitive, so please also try different capitalizations, for example make the first character uppercase.\n\nError: %s</string>
+    <string name="wrong_encryption_password_content">Could not access your account. Most likely you entered the wrong encryption password.\nPlease note that the username is case sensitive, so please also try different capitalizations, for example make the first character uppercase.\n\nError: %s</string>
 
     <string name="signup_title">Enter Signup Details</string>
     <string name="signup_password_restrictions">Password should be at least 8 characters long</string>
 
     <!-- ChangeEncryptionPasswordActivity -->
     <string name="change_encryption_password_title">Change Encryption Password</string>
-    <string name="change_encryption_password_extra_info">Please don\'t use this tool if you believe your encryption password has been compromised. Contact support instead.</string>
+    <string name="change_encryption_password_extra_info">Please don\'t use this tool if you believe someone else could have gained knowledge of your password. Contact support instead.</string>
```

Comment:
If the idea that sharing a password to an entrusted party is to be ruled out, "attained" is possibly more sound.

Reply:
I don't like attained either, and gained knowledge feels clunky. To be honest, I'm starting to think this whole string (even just having it) is confusing. I think I'm going to remove it, and its usage altogether. It's an oddity in the Android client anyway, the rest of the clients don't have it. Any objections or should I go ahead?

Reply:
Please do, it is one of the worst strings. :)

Reply:
Done.